packages/zero-client/src/client/query-manager.ts

```diff
@@ -202,6 +185,34 @@ export class QueryManager {
     };
   }
 
+  update(ast: AST, ttl: TTL) {
```

Comment:
Cool. It seems like this could play a role when queries get stable IDs separate from their hash. Then we could update the AST too.